src/app/static/src/app/components/projects/ShareProject.vue

```diff
@@ -0,0 +1,122 @@
+<template>
+    <div>
+        <a @click="shareProject" href=""
+           v-translate="'share'"></a>
+        <modal :open="open">
+            <h4 v-translate="'shareProject'"></h4>
+            <div v-if="!loading">
+                <div v-html="instructions" id="instructions"></div>
+                <div class="help-text text-muted small">E.g. someone@gmail.com, another@hotmail.com</div>
```

Comment:
I think this will need a French translation - don't think e.g. gets used in French, and could make the email addresses less English